src/layers/temporalconv.jl

```diff
@@ -186,3 +186,42 @@ end
 function Base.show(io::IO, a3tgcn::A3TGCN)
     print(io, "A3TGCN($(a3tgcn.in) => $(a3tgcn.out))")
 end
+
+@doc raw"""
+    TemporalGraphConv(layer)
+
+A constructor for applying to each snapshot of the `TemporalSnapshotsGNNGraph` a convolutional homogeneous graph `layer`.
+
+# Examples 
+
+```jldoctest
+julia> snapshots = [rand_graph(20,40; ndata = rand(3,20)), rand_graph(20,14; ndata = rand(3,20)), rand_graph(20,20; ndata = rand(3,20))];
+
+julia> tsg = TemporalSnapshotsGNNGraph(snapshots);
+
+julia> temp_gcn = TemporalGraphConv(GCNConv(3 => 5, relu))
+TemporalGraphConv(GCNConv(3 => 5, relu))
+
+julia> size(temp_gcn(tsg).ndata.x[1])
+(5, 20)
+```
+"""
+
+struct TemporalGraphConv
+    layer::GNNLayer
+end
+
+Flux.@functor TemporalGraphConv
+
+function (tgconv::TemporalGraphConv)(tsg::TemporalSnapshotsGNNGraph)
+    return TemporalSnapshotsGNNGraph(
+        tsg.num_nodes,
+        tsg.num_edges,
+        tsg.num_snapshots,
+        tgconv.layer.(tsg.snapshots),
+        tsg.tgdata)
+end
+
+function (tgconv::TemporalGraphConv)(tsg::TemporalSnapshotsGNNGraph, x)
+        return tgconv.layer.(tsg.snapshots, x)
```

Comment:
```suggestion
    return tgconv.layer.(tsg.snapshots, x)
```

Also, here broadcasting should be avoided, since both `snapshots` and `x` could be expanded

Reply:
sorry, I got confused, it is the same